specification/README.md

```diff
@@ -11,3 +11,5 @@ The list of Ably Asset Tracking specific error codes.
 | Code | Name | Description |
 | ---- | ---- | ----------- |
 | 100001 | Invalid message | The SDK received a message in an unexpected format. This is treated as a fatal protocol error and the transport will be closed with a failure. |
+| 100002 | Fetching auth token error | The SDK was unable to fetch an auth token. |
```

Comment:
Is this retriable? Given you've gone into depth in the name and description of the error code below this one, it feels like that level of depth should be here also.

Reply:
Good point https://github.com/ably/ably-asset-tracking-common/pull/61/commits/a2ff4f823dcf8112fdc0d1ac9538bdf6bc559275